### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by CharithPramodyaSenanayake** [CharithPramodyaSenanayake/Get-Badges#2] [Q&A] [answered]
What is github

**Comment by PeacebyCHARITH** [comment, marked as answer]
Is an Internet hosting service for software development and version control using Git. It provides the distributed version control of Git plus access control, bug tracking, software feature requests, task management, continuous integration, and wikis for every project.

**Reply by CharithPramodyaSenanayake** [reply]
thanks
